server/eclipse-project/src/main/java/us/freeandfair/corla/query/ExportQueries.java

```diff
@@ -128,24 +128,21 @@ public static void jsonOut(final String query, final OutputStream os) {
     // interleave an object separator (the comma and line break) into the stream
     // of json objects to create valid json thx!
     // https://stackoverflow.com/a/25624818
+    // call .skip() to remove the first separator
     final Stream<Object[]> results =
-        q.stream().flatMap(i -> Stream.of(new String[] {",\n"}, i)).skip(1); // remove
-                                                                             // the
-                                                                             // first
-                                                                             // separator
+            q.stream().flatMap(i -> Stream.of(",\n", i)).skip(1);
 
-    // write json by hand to preserve streaming writes in case of big data
     try {
       os.write("[".getBytes(StandardCharsets.UTF_8));
```

Comment:
Can resultsArr be final, and the for loop changes to iterate over final Object ...? Could push the results.toArray() into the for loop parens I suppose, but it doesn't make a lot of difference.